package.json

```diff
@@ -105,7 +106,7 @@
     "format": "prettier --write \"**/*.+(js|jsx|json|css|md)\"",
     "format:check": "prettier -c \"**/*.+(js|jsx|json|css|md)\"",
     "test": "jest --testPathIgnorePatterns=/browser-test/",
-    "browser-test": "jest browser-test --runInBand",
+    "browser-test": "jest browser-test --runInBand --testRunner=jest-circus/runner",
```

Comment:
Didn't remove the jest-circus runner because it will show a more generic error i.e. the timeout is in the test or is in the hook.